lib/buyers/views.py

```diff
@@ -14,6 +19,13 @@ class BuyerViewSet(NonDeleteModelViewSet):
     serializer_class = BuyerSerializer
     filter_fields = ('uuid', 'active')
 
+    @detail_route(methods=['post'])
```

Comment:
I could. Since the lookup is doing get_object_or_404 theres no real advantage in being in the BuyerViewSet... so only real advantage is that @detail_route is in DRF 3, so when we get upgrade (and delete the @detail_route), this ends up being slightly less code.